cves/CVE-2007-3847.yml

```diff
@@ -168,8 +179,11 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
```

Comment:
Good point about distrusting input! Do you think that the input is complex enough to warrant details under the `complex_inputs` field?